doc/ubuntu.md

```diff
@@ -0,0 +1,51 @@
+# Build from Source on Ubuntu
+
+This document has been written for ubuntu 15.10 but it should work on debian and give a good indication for other systems
+
+## Open Transactions Library
+
+You must have the [open-transactions library installed](https://github.com/Open-Transactions/opentxs-notary/blob/develop/docs/INSTALL-Debian_Ubuntu.txt)
+
+It must be the same branch as MoneyChanger: if you are building the `develop` branch f the MoneyChaner, that is the branch you want to checkout for the Open Transactions library as well
+moneychanger-/moneychanger-qt` 
+## Building MoneyChanger
+
+You can try to build the `master` branch first. If that does not succeed, try the `develop` branch instead.
```

Comment:
I just merged master and develop, so hopefully will cut down on some confusion. You're right though, we should dump master branch for now.